src/neuroglancer/datasource/swc/backend.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {WithParameters} from 'neuroglancer/chunk_manager/backend';
+import {SWCSourceParameters} from 'neuroglancer/datasource/swc/base';
+import {SkeletonChunk, SkeletonSource} from 'neuroglancer/skeleton/backend';
+import {decodeSwcSkeletonChunk} from 'neuroglancer/skeleton/decode_swc_skeleton';
+import {CancellationToken} from 'neuroglancer/util/cancellation';
+import {cancellableFetchOk, responseArrayBuffer} from 'neuroglancer/util/http_request';
+import {registerSharedObject} from 'neuroglancer/worker_rpc';
+
+@registerSharedObject() export class SWCSkeletonSource extends
+(WithParameters(SkeletonSource, SWCSourceParameters)) {
+  download(chunk: SkeletonChunk, cancellationToken: CancellationToken) {
+    const {parameters} = this;
+    let bodyid = `${chunk.objectId}`;
+    // Change the url pattern for SWC
+    // const url = `${parameters.baseUrl}/api/node/${parameters['nodeKey']}` +
+    //     `/${parameters['dataInstanceKey']}/key/` + bodyid + '_swc';
+    const url = `${parameters.baseUrl}/swc/` + bodyid ;
```

Comment:
I recommend that you support data source URLS like:

"https://whatever/path/{id}.swc"  and then here you would replace {id} with the actual id converted to a string, e.g. via String.prototype.replace.